Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_usb_wfssrv.cpp

```diff
@@ -0,0 +1,266 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/IPC_HLE/WII_IPC_HLE_Device_usb_wfssrv.h"
+
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/FileUtil.h"
+#include "Common/Logging/Log.h"
+#include "Common/NandPaths.h"
+#include "Core/HW/DVDInterface.h"
+#include "Core/HW/Memmap.h"
+
+u32 num_calls = 0;
```

Comment:
Oh, I completely forgot about that, was meaning to actually implement directory listing before getting this merged, heh. I think at this point I'll just make the stubbing a bit nicer and have some big warnings.